File: SampleSetup/Output/PHP/SampleProject/Protocol/Model/ApiAddress.php
<?php
namespace SampleProject\Protocol\Model;

class ApiAddress extends \SampleProject\Protocol\Model\AbstractModelWithId {
    /** @var string */ public $CountryId;

    /** @var \SampleProject\Protocol\Model\ApiCountry */ public $Country;

    /** @var string */ public $CityId;

    /** @var \SampleProject\Protocol\Model\ApiCity */ public $City;

    /** @var string */ public $PostalCode;

    /** @var string */ public $AddressLine1;

    /** @var string */ public $AddressLine2;

    /** @var float|null */ public $GeoLat;

    /** @var float|null */ public $GeoLon;

    /** @var string */ public $Description;
}

File: SampleSetup/Output/Kotlin/SampleProject/Protocol/Model/ApiAddress.kt
package SampleProject.Protocol.Model

import SampleProject.Protocol.Model.*
open class ApiAddress: AbstractModelWithId() {
    var CountryId: String

    var Country: ApiCountry

    var CityId: String

    var City: ApiCity

    var PostalCode: String

    var AddressLine1: String

    var AddressLine2: String

    var GeoLat: Double?

    var GeoLon: Double?

    var Description: String
}
